HeroVault/HeroVault/Features/Home/HomeScreen/HomeScreenViewController.swift

```diff
@@ -51,6 +53,14 @@ class HomeScreenViewController: UIViewController {
         listTableView.dataSource = self
         listTableView.delegate = self
     }
+    
+    private func setUpSegmentedControl() {
+        var selectedSegmentAttributes: [NSAttributedString.Key: Any] = [:]
```

Comment:
Can be simplified to
let selectedSegmentAttributes: [NSAttributedString.Key: Any] = [.foregroundColor: UIColor.white]
segmentedControl.backgroundColor = .white
segmentedControl.tintColor = .primary
segmentedControl.setTitleTextAttributes(selectedSegmentAttributes, for: .selected)